### DISCUSSION TITLE
Windows server Web console

### DISCUSSION THREAD

**Discussion by Jim-Crosby** [rustdesk/rustdesk#5280] [Q&A] [answered]
Is it possible to install the Web Admin Console on Windows Server, I've installed HHBR and HBBS as services and while I can view logs etc which shows that they are working. 
I just can't find how to actually install the Web Console, Any help/guidance would be really appreciated.

**Comment by dinger1986** [comment, marked as answer]
Are you using RustDesk Server Pro?

If not there is no web console with the opensource version.

**Reply by Jim-Crosby** [reply]
Thanks for the Info I'll get a licence,
